src/V3EmitMk.cpp

```diff
@@ -224,6 +228,10 @@ class EmitMk {
         of.puts("\n### Default rules...\n");
         of.puts("# Include list of all generated classes\n");
         of.puts("include " + v3Global.opt.prefix() + "_classes.mk\n");
+        if (v3Global.opt.hierTop()) {
+            of.puts("# Include rules for hierarchy blocks\n");
```

Comment:
```suggestion
            of.puts("# Include rules for hierarchy blocks\n");
```

Here and in stats where you say "hierarchy blocks" I suggest you say "--hierarchical" to make it a bit more specific that this is related to the flag, versus hierarchy as a general concept.

Reply:
3cade714ea6fb4fa9e092557f05686d7f6d809b7 uses "hierarchical" in the hierarchical Verilation context.